### ISSUE TITLE
[inductor] IMA - maximum, accuracy for `rexnet_100`

### ISSUE THREAD

**Issue by anijain2305** [pytorch/torchdynamo#1460]
Repro
~~~

import torch
from torch import tensor, device
import torch.fx as fx
from torchdynamo.testing import rand_strided
from math import inf
from torch.fx.experimental.proxy_tensor import make_fx

# torch version: 1.13.0a0+git071f875
# torch cuda version: 11.6
# torch git version: 071f875046202b87213865dfc180abdf8368f116


# CUDA Info: 
# nvcc: NVIDIA (R) Cuda compiler driver 
# Copyright (c) 2005-2022 NVIDIA Corporation 
# Built on Thu_Feb_10_18:23:41_PST_2022 
# Cuda compilation tools, release 11.6, V11.6.112 
# Build cuda_11.6.r11.6/compiler.30978841_0 

# GPU Hardware Info: 
# NVIDIA A100-SXM4-40GB : 8 


from torch.nn import *
class Repro(torch.nn.Module):
    def __init__(self):
        super().__init__()

    
    
    def forward(self, mul_192, lift_fresh_copy_30):
        maximum_15 = torch.ops.aten.maximum.default(mul_192, lift_fresh_copy_30);  mul_192 = lift_fresh_copy_30 = None
        return (maximum_15,)
        
args = [((2, 1044, 7, 7), (51156, 49, 7, 1), torch.float32, 'cuda'), ((), (), torch.float32, 'cpu')]
args = [rand_strided(sh, st, dt, dev) for (sh, st, dt, dev) in args]
mod = make_fx(Repro())(*args)

from torchinductor.compile_fx import compile_fx_inner
from torchdynamo.debug_utils import same_two_models

ref = mod(*args)
print(ref[0].sum())

compiled = compile_fx_inner(mod, args)
res = compiled(*args)
print(res[0].sum())

~~~

**Comment by anijain2305** [comment]
Also silently leads to bad accuracy errors - 
~~~

import torch
from torch import tensor, device
import torch.fx as fx
from torchdynamo.testing import rand_strided
from math import inf
from torch.fx.experimental.proxy_tensor import make_fx

# torch version: 1.13.0a0+git071f875
# torch cuda version: 11.6
# torch git version: 071f875046202b87213865dfc180abdf8368f116


# CUDA Info: 
# nvcc: NVIDIA (R) Cuda compiler driver 
# Copyright (c) 2005-2022 NVIDIA Corporation 
# Built on Thu_Feb_10_18:23:41_PST_2022 
# Cuda compilation tools, release 11.6, V11.6.112 
# Build cuda_11.6.r11.6/compiler.30978841_0 

# GPU Hardware Info: 
# NVIDIA A100-SXM4-40GB : 8 


from torch.nn import *
class Repro(torch.nn.Module):
    def __init__(self):
        super().__init__()
        self.register_buffer('_tensor_constant30', torch.randn([], dtype=torch.float32))

    
    
    def forward(self, arg219_1, arg220_1, arg221_1, arg222_1, add_127, convolution_71, _to_copy_118, add_128):
        sqrt_59 = torch.ops.aten.sqrt.default(add_128);  add_128 = None
        reciprocal_59 = torch.ops.aten.reciprocal.default(sqrt_59);  sqrt_59 = None
        mul_189 = torch.ops.aten.mul.Tensor(reciprocal_59, 1);  reciprocal_59 = None
        unsqueeze_472 = torch.ops.aten.unsqueeze.default(_to_copy_118, -1);  _to_copy_118 = None
        unsqueeze_473 = torch.ops.aten.unsqueeze.default(unsqueeze_472, -1);  unsqueeze_472 = None
        unsqueeze_474 = torch.ops.aten.unsqueeze.default(mul_189, -1);  mul_189 = None
        unsqueeze_475 = torch.ops.aten.unsqueeze.default(unsqueeze_474, -1);  unsqueeze_474 = None
        sub_59 = torch.ops.aten.sub.Tensor(convolution_71, unsqueeze_473);  convolution_71 = unsqueeze_473 = None
        mul_190 = torch.ops.aten.mul.Tensor(sub_59, unsqueeze_475);  sub_59 = unsqueeze_475 = None
        unsqueeze_476 = torch.ops.aten.unsqueeze.default(arg219_1, -1);  arg219_1 = None
        unsqueeze_477 = torch.ops.aten.unsqueeze.default(unsqueeze_476, -1);  unsqueeze_476 = None
        unsqueeze_478 = torch.ops.aten.unsqueeze.default(arg220_1, -1);  arg220_1 = None
        unsqueeze_479 = torch.ops.aten.unsqueeze.default(unsqueeze_478, -1);  unsqueeze_478 = None
        mul_191 = torch.ops.aten.mul.Tensor(mul_190, unsqueeze_477);  mul_190 = unsqueeze_477 = None
        add_129 = torch.ops.aten.add.Tensor(mul_191, unsqueeze_479);  mul_191 = unsqueeze_479 = None
        relu_12 = torch.ops.aten.relu.default(add_129);  add_129 = None
        convolution_72 = torch.ops.aten.convolution.default(relu_12, arg221_1, arg222_1, [1, 1], [0, 0], [1, 1], False, [0, 0], 1);  relu_12 = arg221_1 = arg222_1 = None
        sigmoid_12 = torch.ops.aten.sigmoid.default(convolution_72);  convolution_72 = None
        mul_192 = torch.ops.aten.mul.Tensor(add_127, sigmoid_12);  add_127 = sigmoid_12 = None
        _tensor_constant30 = self._tensor_constant30
        lift_fresh_copy_30 = torch.ops.aten.lift_fresh_copy.default(_tensor_constant30);  _tensor_constant30 = None
        maximum_15 = torch.ops.aten.maximum.default(mul_192, lift_fresh_copy_30);  mul_192 = lift_fresh_copy_30 = None
        return (maximum_15,)
        
args = [((87,), (1,), torch.float32, 'cuda'), ((87,), (1,), torch.float32, 'cuda'), ((1044, 87, 1, 1), (87, 1, 1, 1), torch.float32, 'cuda'), ((1044,), (1,), torch.float32, 'cuda'), ((2, 1044, 7, 7), (51156, 49, 7, 1), torch.float32, 'cuda'), ((2, 87, 1, 1), (87, 1, 1, 1), torch.float32, 'cuda'), ((87,), (1,), torch.float32, 'cuda'), ((87,), (1,), torch.float32, 'cuda')]
args = [rand_strided(sh, st, dt, dev) for (sh, st, dt, dev) in args]
mod = make_fx(Repro())(*args)

from torchinductor.compile_fx import compile_fx_inner
from torchdynamo.debug_utils import same_two_models

compiled = compile_fx_inner(mod, args)
assert same_two_models(mod, compiled, args, only_fwd=True), "Accuracy failed"

~~~

**Comment by yanboliang** [comment]
Verified this has been fixed by https://github.com/pytorch/torchdynamo/pull/1284
